src/TestData/Commands/DonorSeedCommand.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace GiveTestData\TestData\Commands;
+
+use WP_CLI;
+use GiveTestData\TestData\Factories\DonorFactory;
+use GiveTestData\TestData\Repositories\DonorRepository;
+
+/**
+ * Class DonorSeedCommand
+ * @package GiveTestData\TestData\Commands
+ *
+ * A WP-CLI command for seeding donors.
+ */
+class DonorSeedCommand {
+	/**
+	 * @var DonorFactory
+	 */
+	private $donorFactory;
+	/**
+	 * @var DonorRepository
+	 */
+	private $donorRepository;
+
+	/**
+	 * @param DonorFactory $donorFactory
+	 * @param DonorRepository $donorRepository
+	 */
+	public function __construct( DonorFactory $donorFactory, DonorRepository $donorRepository ) {
```

Comment:
I'm noticing some inconsistencies in how the classes are formatted - might want to look into the Linter.